packages/lit-dev-content/site/home/5-who-is-using.html

```diff
@@ -21,6 +21,18 @@ <h1>
         </div>
       </a>
 
+      <a target="_blank" rel="noopener" href="https://reddit.com">
+        <lazy-svg
+          svg-role="img"
+          label="Reddit"
+          href="{{ site.baseurl }}/images/logos/reddit.svg#full"
+          loading="visible">
+        </lazy-svg>
+        <div>
+          <span>Frontend UI</span>
+        </div>
+      </a>
+
```

Comment:
Nit: I think the rest of the companies here are all alphabetically ordered, except JSTOR which was added to the end. Should it all be re-ordered?

Reply:
Good point, WDYT @justinfagnani

Reply:
alphabetized!